deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Bail on unset variables, errors and trace execution
+set -eux
+
+
+# Set up Docker image
+#####################
+
+cat <<EOF > .dockerignore
+.DS_Store
+.git
+.github
+.serverless
+.webpack
+cypress
+dist
+node_modules
+tmp
+EOF
+
+cat <<EOF > Dockerfile
+FROM node:22
+COPY . /build
+WORKDIR /build
+RUN npm ci
+EOF
+
+dockerTag=cmr-stac-eudoro-$STAGE_NAME
+stageOpts="--stage $STAGE_NAME "
+
+docker build -t $dockerTag .
+
+# Convenience function to invoke `docker run` with appropriate env vars instead of baking them into image
+dockerRun() {
```

Comment:
Currently deployments are done via `bamboo-specs` on CMR stac will need to update that repo to put these new variables or otherwise remove `cmr-stac` from the bamboo specs deployment